client/src/components/ApplicantsList.js

```diff
@@ -48,10 +75,17 @@ const ApplicantsList = () => {
               </Button>
             </div>
 
-            <div className="button-style">
-              <Button fluid primary>
-                Contact Applicant
-              </Button>
+            <div key={key + "12d"} className="button-style">
+              <ContactApplicantButton
+                profilePublicId={profile.profile_public_id}
+                to={`/public-applicant-profiles/${profile.profile_public_id}`}
+                isDisabled={messages.some(
+                  message =>
+                    message.profile_public_id === profile.profile_public_id
```

Comment:
Sorry Andrei, I forgot to check this line, it has to check also the recruiter ID is matching so when signing in a different recruiter will not see the same list of applicants that contacted by a different recruiter. 
e.g `message. recruiter_id === userId && message.profile_public_id === profile.profile_public_id`